### PR TITLE
Page options menu

### PR DESCRIPTION
**Pull request by ecotopian**
@vauxhall this is still wip, but I already wanted to gather your input before your leave as it's quite a bit 😄

### CODE REVIEW

File: Client/Ecosia/UI/PageAction/PageActionMenu.swift [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+        return UITableView.automaticDimension
+    }
+
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return viewModel.actions.count
+    }
+
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return viewModel.actions[section].count
+    }
```
Comment (vauxhall) [inline Client/Ecosia/UI/PageAction/PageActionMenu.swift:107]:
```suggestion
    }

```

Comment (vauxhall) [reply Client/Ecosia/UI/PageAction/PageActionMenu.swift:107]: Just a small decoration

File: Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheet.swift [hunk 2]
```diff
@@ -365,11 +365,11 @@ extension PhotonActionSheet: UITableViewDataSource, UITableViewDelegate {
     func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
         guard let section = viewModel.actions[safe: indexPath.section],
               let action = section[safe: indexPath.row],
-              let custom = action.items[0].customHeight
```
Comment (vauxhall) [inline Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheet.swift:368]: This definitely seems like a good improvement and safer too 👍

**Review by vauxhall — APPROVED** [review]
Already looking great! 💯

**Comment by ecotopian** [comment]
> It looks great, despite the tests failing 🚀

fixed! :)

**Pull request merged**